### PR TITLE
Sidebar

### PR DESCRIPTION
**Pull request by rizgararab**

### CODE REVIEW

**Review by jp-2019 — CHANGES_REQUESTED** [review]
move code from css file to html code. 
extra spacing must be removed. 
code isn't running.

File: angular-techDraft/src/app/top-nav-bar/top-nav-bar.component.html [hunk 1]
```diff
@@ -14,3 +14,9 @@
     </div>
   </div>
 </div>
+
```
Comment (jp-2019) [inline angular-techDraft/src/app/top-nav-bar/top-nav-bar.component.html]: remove extra spacing

**Review by jp-2019 — COMMENTED** [review]
need menu button image (need to add image asset)

**Review by jp-2019 — APPROVED** [review]

**Review by JaredHook — CHANGES_REQUESTED** [review]
pull request is looking to delete integral files

**Review by jp-2019 — DISMISSED** [review]
what's going on with the delete files??

**Comment by shakirullah7892** [comment]
I agree with Jared and Jeaniie

File: angular-techDraft/src/app/top-nav-bar/top-nav-bar.component.html (hunk 1 above)
Comment (rizgararab) [reply angular-techDraft/src/app/top-nav-bar/top-nav-bar.component.html]: extra spacing removed

**Review by rizgararab — COMMENTED** [review]
changes has been done

File: angular-techDraft/src/app/app.module.ts [hunk 2]
```diff
@@ -9,11 +9,11 @@ import { GgithabButtonComponent } from './ggithab-button/ggithab-button.componen
 import { ShaksCallToActionCompComponent } from './shaks-call-to-action-comp/shaks-call-to-action-comp.component';
 import { BottomComponent } from './bottom/bottom.component';
 import { ThemeComponent } from './theme/theme.component';
+import { SidebarComponent } from './sidebar/sidebar.component';
 import { TeamTableComponent } from './team-table/team-table.component';
 import { MyNavComponent } from './my-nav/my-nav.component';
 import { HomePageComponent } from './home-page/home-page.component';
 
-
```
Comment (jp-2019) [inline angular-techDraft/src/app/app.module.ts:16]: -?

File: angular-techDraft/src/app/ggithab-button/ggithab-button.component.html [hunk 3]
```diff
@@ -1,7 +1,9 @@
 <div class="item">
+
```
Comment (shakirullah7892) [inline angular-techDraft/src/app/ggithab-button/ggithab-button.component.html]: extra spacing has to be removed.

File: angular-techDraft/src/app/ggithab-button/ggithab-button.component.html [hunk 4]
```diff
@@ -1,7 +1,9 @@
...
   <script type="text/javascript">
     window.twttr = (function (d, s, id) { var t, js, fjs = d.getElementsByTagName(s)[0]; if (d.getElementById(id)) { return } js = d.createElement(s); js.id = id; js.src = "https://platform.twitter.com/widgets.js"; fjs.parentNode.insertBefore(js, fjs); return window.twttr || (t = { _e: [], ready: function (f) { t._e.push(f) } }) }(document, "script", "twitter-wjs"));
   </script>
   <iframe class="github" src="//ghbtns.com/github-btn.html?user=semantic-org&amp;repo=semantic-ui&amp;type=watch&amp;count=true" allowtransparency="true" scrolling="0" width="100" height="20" frameborder="0"></iframe>
+
```
Comment (shakirullah7892) [inline angular-techDraft/src/app/ggithab-button/ggithab-button.component.html]: extra spacing has to be removed here too.

File: angular-techDraft/src/app/newsletterform/newsletterform.component.css [hunk 5]
```diff
@@ -6,6 +6,7 @@
   width: auto;
 }
 
+
```
Comment (shakirullah7892) [inline angular-techDraft/src/app/newsletterform/newsletterform.component.css]: extra line needs to removed.

File: angular-techDraft/src/app/newsletterform/newsletterform.component.html [hunk 6]
```diff
@@ -19,3 +19,4 @@ <h3 class="ui header">
     </div>
   </div>
 </div>
+
```
Comment (shakirullah7892) [inline angular-techDraft/src/app/newsletterform/newsletterform.component.html]: extra line needs to be removed here too.

File: angular-techDraft/src/app/sidebar/sidebar.component.spec.ts [hunk 7]
```diff
@@ -1,20 +1,20 @@
...
-  let component: LoginComponent;
-  let fixture: ComponentFixture<LoginComponent>;
+describe('SidebarComponent', () => {
+  let component: SidebarComponent;
+  let fixture: ComponentFixture<SidebarComponent>;
 
   beforeEach(async(() => {
     TestBed.configureTestingModule({
-      declarations: [ LoginComponent ]
+      declarations: [ SidebarComponent ]
```
Comment (shakirullah7892) [inline angular-techDraft/src/app/sidebar/sidebar.component.spec.ts:11]: All of the sidebar spect.ts has to be reverted.

**Comment by rizgararab** [comment]
All approved by Shak

**Comment by rizgararab** [comment]
Approved

**Review by JaredHook — APPROVED** [review]

**Review by JaredHook — APPROVED** [review]

**Review by JaredHook — APPROVED** [review]

**Review by shakirullah7892 — APPROVED** [review]

**Pull request merged**